### PR TITLE
KOGITO-8293: Add tests for basic functions of SWF Web Tools

### PR DESCRIPTION
**Pull request by tomasdavidorg**
https://issues.redhat.com/browse/KOGITO-8293

### CODE REVIEW

**Review by paulovmr — APPROVED** [review]

File: packages/serverless-logic-sandbox/it-tests/e2e/UploadFiles.cy.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+describe("Serverless Logic Web Tools - Upload files test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should upload dashbuider file and check", () => {
```
Comment (barboras7) [inline packages/serverless-logic-sandbox/it-tests/e2e/UploadFiles.cy.ts]:
```suggestion
  it("should upload dashbuider file and check rendered content", () => {
```

**Review by caponetto — APPROVED** [review]

File: packages/serverless-logic-sandbox/it-tests/e2e/UploadFiles.cy.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    // switch to YAML file
+    cy.ouia({ ouiaId: "file-type-label" }).click();
+    cy.get("a[href$='/file/helloYamlWorkflow.sw.yaml']").click();
+
+    // wait until file is switched
+    cy.get("a[download]").eq(0).should("have.attr", "download", "helloYamlWorkflow.sw.yaml");
+    cy.loadEditor();
+
+    // check header labels
```
Comment (barboras7) [inline packages/serverless-logic-sandbox/it-tests/e2e/UploadFiles.cy.ts]:
```suggestion
    // check YAML header labels
```

File: packages/serverless-logic-sandbox/it-tests/e2e/UploadFiles.cy.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+  it("should upload 2 files (JSON and YAML) and check", () => {
+    // upload JSON and YAML files
+    cy.get("#upload-field").attachFile(
+      ["uploadFile/helloJsonWorkflow.sw.json", "uploadFile/helloYamlWorkflow.sw.yaml"],
+      { subjectType: "drag-n-drop" }
+    );
+    cy.loadEditor();
+
+    // check header labels
```
Comment (barboras7) [inline packages/serverless-logic-sandbox/it-tests/e2e/UploadFiles.cy.ts]:
```suggestion
    // check JSON header labels
```

File: packages/serverless-logic-sandbox/it-tests/e2e/UploadFiles.cy.ts (hunk 1 above)
Comment (barboras7) [reply packages/serverless-logic-sandbox/it-tests/e2e/UploadFiles.cy.ts]: Please, provide some information about what is being checked (both here and below at line 44)

**Review by barboras7 — CHANGES_REQUESTED** [review]

**Review by barboras7 — APPROVED** [review]

**Pull request merged**